tests/fuzz/test-afl.sh

```diff
@@ -4,10 +4,15 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+echo "Usage: $0 <seconds to run> <bit width>"
+
+# export AFL_USE_ASAN=1
```

Comment:
Should these be here? If not, should we add a comment explaining why they are here and what they can do?

Reply:
Added comment